Framework/API/inc/MantidAPI/NexusFileLoader.h

```diff
@@ -0,0 +1,34 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2013 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "MantidAPI/IFileLoader.h"
+#include "MantidKernel/NexusHDF5Descriptor.h"
+#include "MantidKernel/System.h"
+
+namespace Mantid::API {
+
+class DLLExport NexusFileLoader
```

Comment:
`DLLExport` -> `MANTID_API_DLL`. You'll need to swap `#include "MantidKernel/System.h"` for `#include "MantidAPI/DllConfig.h"`